torch/csrc/stable/tensor.h

```diff
@@ -109,6 +109,20 @@ class Tensor {
     return device_type == aoti_torch_device_type_cuda();
   }
 
+  bool is_cpu() const {
```

Comment:
Let's land this PR with just this wrapper API (move the dtype one out). Also add testing please! Following the examples of other PRs touching this file!

Reply:
Will do!